### PR TITLE
[Merged by Bors] - refactor(MetricSpace/Basic): use `cobounded` more

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/MetricSpace/Basic.lean [hunk 1]
```diff
@@ -2833,32 +2861,32 @@ def evalDiam : PositivityExt where eval {_ _} _zα _pα e := do
...
 end Mathlib.Meta.Positivity
 
-theorem comap_dist_right_atTop_le_cocompact (x : α) :
-    comap (fun y => dist y x) atTop ≤ cocompact α := by
-  refine' Filter.hasBasis_cocompact.ge_iff.2 fun s hs => mem_comap.2 _
-  rcases hs.isBounded.subset_closedBall x with ⟨r, hr⟩
-  exact ⟨Ioi r, Ioi_mem_atTop r, fun y hy hys => (mem_closedBall.1 <| hr hys).not_lt hy⟩
-#align comap_dist_right_at_top_le_cocompact comap_dist_right_atTop_le_cocompact
+#align comap_dist_right_at_top_le_cocompact Metric.cobounded_le_cocompactₓ
+#align comap_dist_left_at_top_le_cocompact Metric.cobounded_le_cocompactₓ
```
Comment (sgouezel) [inline Mathlib/Topology/MetricSpace/Basic.lean:2865]: Could you move these right after the theorem they mention.

**Review by sgouezel — COMMENTED** [review]
LGTM, thanks!
bors d+

**Comment by urkud** [comment]
bors merge

**Comment by ADedecker** [comment]
bors retry

**Pull request closed without merging**